graylog2-server/src/main/java/org/graylog2/bootstrap/preflight/web/resources/PreflightResource.java

```diff
@@ -106,12 +111,40 @@ public CA get() throws KeyStoreStorageException {
         return caService.get();
     }
 
+    @GET
+    @Path("/ca/certificate")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getCaPublicKey() {
```

Comment:
I am a little bit confused here...
Endpoint path is `/ca/certificate`, method name is `getCaPublicKey`, the code seems to encode the certificate object, not public key.

I may be wrong, but maybe the naming needs to be unified on all those levels?

Reply:
you are right, fixed